modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -1,11 +1,13 @@
-= Create and Publish a RAML API Fragment in the Text Editor in API Designer
+= Create and Publish an API Fragment in the Text Editor in API Designer
 
-You can create API fragments directly in RAML with the help of code suggestions that appear within the text editor. 
+You can create API fragments directly in RAML, OAS 3.0, and JSON Schema with the help of code suggestions that appear within the text editor. 
```

Comment:
I'm not sure if this is related, but above is "API fragments (RAML), OAS Components, JSON schemas," Can fragments be any of these types?